indexer/src/main/java/juice/modules/ConsumerModule.java

```diff
@@ -0,0 +1,34 @@
+package juice.modules;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import config.ConfigFileFinder;
+import config.ConfigurationFactory;
+import config.ConsumerConfiguration;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+
+import java.util.Properties;
+
+public class ConsumerModule extends AbstractModule {
```

Comment:
`KafkaConsumerModule` is more appropriate name

Reply:
done